tests/fsharpqa/testenv/src/NugetTestPackages/buildfscnuget.bat

```diff
@@ -1,12 +1,12 @@
 @echo off
 
-set pkgversion="1.0.0-alpha-0003"
+set pkgversion="1.0.0-alpha-0006"
 
-:: Check prerequisites
-if not '%VisualStudioVersion%' == '' goto vsversionset
-if exist "%ProgramFiles(x86)%\Microsoft Visual Studio 14.0\common7\ide\devenv.exe" set VisualStudioVersion=14.0
-if exist "%ProgramFiles%\Microsoft Visual Studio 14.0\common7\ide\devenv.exe" set VisualStudioVersion=14.0
-if exist "%VS140COMNTOOLS%" set VisualStudioVersion=14.0
+:: Check prerequisites if not '%VisualStudioVersion%' == '' goto vsversionset if
+exist "%ProgramFiles(x86)%\Microsoft Visual Studio  14.0\common7\ide\devenv.exe"
+set VisualStudioVersion=14.0  if exist  "%ProgramFiles%\Microsoft Visual  Studio
+14.0\common7\ide\devenv.exe"    set     VisualStudioVersion=14.0    if     exist
```

Comment:
The files is actually redundant I think.  All of the "real work" is now commented out.

I will delete the file.